component/main.jsonnet

```diff
@@ -8,23 +8,22 @@ local inv = kap.inventory();
 local params = inv.parameters.appuio_cloud;
 
 local isOpenshift = std.startsWith(inv.parameters.facts.distribution, 'openshift');
-local monitoringLabel =
-  if isOpenshift then
-    {
-      'openshift.io/cluster-monitoring': 'true',
-    }
-  else
-    {
-      SYNMonitoring: 'main',
-    };
+local namespaceLabels = (
+  if isOpenshift then {'openshift.io/cluster-monitoring': 'true'}
+  else {'SYNMonitoring': 'main'}
+) + params.namespaceLabels;
+local namespaceAnnotations = (
+  if isOpenshift then {'openshift.io/node-selector': ''}
+  else {}
+) + params.namespaceAnnotations;
```

Comment:
not sure why we even bother with openshift/non-openshift here, but doesn't really matter.